pkg/client/clients.go

```diff
@@ -0,0 +1,199 @@
+package client
```

Comment:
@praveen4g0 , if the file is copied from `pipelines`, can't we not use it through `go modules`/`vendor` instead of making a copy?

Reply:
@sthaha No it's not Copy of Clients from ```Pipelines``` , and Also ```clients``` from Pipelines are not exportable, Reason I have come up with this clients.go file is, to install an operator, validate it on any Cluster (OCP), instead of using ```operator-sdk local```